reference.md

```diff
@@ -161,6 +163,8 @@ admit lists, tuples only admit tuples, bools only admit bools.
 * A list type `A` admits another list type `B` iff `A.max-len >= B.max-len` and
   `A.entry-type` admits `B.entry-type`.
 * A buffer type `A` admits another buffer type `B` iff `A.max-len >= B.max-len`.
+* An ascii string type `A` admits another ascii string type `B` iff `A.max-len >= B.max-len`.
+* A utf string type `A` admits another utf string type `B` iff `A.max-len >= B.max-len`.
```

Comment:
```suggestion
* A utf-8 string type `A` admits another utf-8 string type `B` iff `A.max-len >= B.max-len`.
```

Reply:
@friedger have you seen this suggestion?